recipes/libblitz/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "0.10" %}
+
+package:
+  name: libblitz
+  version: {{ version }}
+
+source:
+  fn: blitz-{{ version }}.tar.gz
+  url: http://downloads.sourceforge.net/project/blitz/blitz/Blitz%2B%2B%20{{ version }}/blitz-{{ version }}.tar.gz
+  md5: 66268b92bda923735f2e3afc87dcb58a
+
+build:
+  number: 0
+  skip: true   # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - gcc
```

Comment:
Do we need `gcc`? On Linux we have gcc (4.4?) at present. It will eventually switch to gcc 5 in backwards compatibility mode. On Mac, we prefer clang.